commitizen/git.py

```diff
@@ -44,13 +44,20 @@ def __eq__(self, other) -> bool:
 
 class GitCommit(GitObject):
     def __init__(
-        self, rev, title, body: str = "", author: str = "", author_email: str = ""
+        self,
+        rev,
+        title,
+        body: str = "",
+        author: str = "",
+        author_email: str = "",
+        parents: list[str] = [],
```

Comment:
You can't use a list as default value, it will persist between calls, use an optional list instead.
```suggestion
        parents: list[str] | None = None,
```

Reply:
Applied the suggestion.